MANIFEST.in

```diff
@@ -3,6 +3,9 @@ include README.md
 exclude .pre-commit-config.yaml
 
 recursive-include brainglobe_workflows *.py
+include brainglobe_workflows/cellfinder/default_config.json
+
+recursive-exclude brainglobe_workflows/cellfinder/notebooks *.py
```

Comment:
I wanted to have them in the history for my own reference (so I was not gitingoring them) but I didn't want them to have them in the sdist. But I am not sure if this is best practice / the best way to do this 🤔 

I just removed this line for now because I had actually deleted the notebooks a while ago, but any advice on how to do this better is more than welcome!